shopping_mall/backend/tests/eval/test_cs_response_quality.py

```
@@ -0,0 +1,282 @@
+
+    total_checks = 0
+    passed_checks = 0
+    case_results: list[dict] = []
+
+    for case in QUALITY_CASES:
+        print(f"\n[Case {case.id}] {case.description}")
+        print(f"  쿼리: \"{case.query}\"")
+
+        try:
+            answer, elapsed = call_chatbot(opener, case.query, case.session_id)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)